xledger_combined/metadata.json

```diff
@@ -0,0 +1 @@
+{"key":"xledger_combined","authentication":"CUSTOM"}
```

Comment:
@GuyVanWert Is there a reason this is called xledger_combined? Would the key xledger make more sense here?